lib/logger/config-serializer.ts

```diff
@@ -1,8 +1,7 @@
-const traverse = require('traverse');
+import traverse from 'traverse';
+import { RenovateConfig } from '../config/common';
 
-module.exports = configSerializer;
-
-function configSerializer(config) {
+export default function(config: RenovateConfig): RenovateConfig {
```

Comment:
```suggestion
export default function configSerializer(config: RenovateConfig): RenovateConfig {
```

Shouldn't we add a name? It will be shown in stack traces if any error occure.